Library/Homebrew/cmd/update-report.rb

```diff
@@ -229,6 +240,7 @@ def report
         rescue Exception => e # rubocop:disable Lint/RescueException
           onoe "#{e.message}\n#{e.backtrace.join "\n"}" if Homebrew::EnvConfig.developer?
         end
+
         @report[:M] << tap.formula_file_to_name(src)
```

Comment:
```suggestion
```

Reply:
I think moving this would result in the version check not having any effect.

There is some argument however that the version check stuff should be moved to where the list is printed, so that `DescriptionCacheStore` updates don't depend on version bumps.

Reply:
> I think moving this would result in the version check not having any effect.

Yeh, sorry, missed that.